guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -34,8 +34,8 @@ Ensure that you understand the following terms:
 
 To clone {ProjectServer}, ensure that you have the following resources available:
 
-* A minimal install of {RHELServer} 7 to become the target server.
-Do not install {RHEL} 7 software groups, or third-party applications.
+* A minimal install of {RHELServer} 8 to become the target server.
```

Comment:
There is no `{RHELServer} 8` since RHEL 8 dropped the distinction between server and workstation editions
```suggestion
* A minimal install of {RHEL} 8 to become the target server.
```